src/app/api/v1/user/update-comm-option/route.ts

```diff
@@ -19,24 +20,34 @@ import {
 export type EmailUpdateCommTypeOfOptions = "-1" | "0" | "1";
 
 export interface EmailUpdateCommOptionRequest {
-  communicationOption: EmailUpdateCommTypeOfOptions;
+  instantBreachAlerts: EmailUpdateCommTypeOfOptions;
+  monthlyMonitorReport: boolean;
 }
 
 export async function POST(req: NextRequest) {
   const token = await getToken({ req });
 
   if (typeof token?.subscriber?.fxa_uid === "string") {
     try {
-      const { communicationOption }: EmailUpdateCommOptionRequest =
-        await req.json();
+      const {
+        instantBreachAlerts,
+        monthlyMonitorReport,
+      }: EmailUpdateCommOptionRequest = await req.json();
       const subscriber = await getSubscriberByFxaUid(token.subscriber?.fxa_uid);
       if (!subscriber) {
         throw new Error("No subscriber found for current session.");
       }
+      // -1 = Set as null, do not send instant notifications. Newly added in MNTOR-1368
       // 0 = Send breach alerts to the corresponding affected emails.
       // 1 = Send all breach alerts to user's primary email address.
```

Comment:
Would it be an idea to use string constants here, instead of numbers? (Possibly with a temporary fallback to also accept `0` and `1` to accommodate pods running different code during deployment, though that's probably overkill.)

Reply:
Good idea, changed to strings